spec/shell_spec.cr

```diff
@@ -52,7 +52,7 @@ Spec2.describe "Shell" do
 
             it "stderr not empty" { expect(subject.stderr).not_to eq "" }
             it "stderr contains error message" {
-              expect(subject.stderr).to match(/command not found/)
+              expect(subject.stderr).to match(/not found/)
```

Comment:
Here, the error message depends on shell. This will relax restrictions and make pass a test on travis-ci.